projects/igniteui-angular/src/lib/grids/columns/column.component.ts

```diff
@@ -569,6 +570,11 @@ export class IgxColumnComponent implements AfterContentInit, OnDestroy {
     @WatchColumnChanges()
     @Input()
     formatter: (value: any) => any;
+
+    @notifyChanges()
+    @WatchColumnChanges()
+    @Input()
+    summaryFormatter: (summary: IgxSummaryResult, summaryOperand: IgxSummaryOperand) => string;
```

Comment:
Let's make the return type `any` in case they want to format the values in other types.
```suggestion
    summaryFormatter: (summary: IgxSummaryResult, summaryOperand: IgxSummaryOperand) => any;
```